models/POP/shell_scripts/cesm2_1/DART_params.csh

```diff
@@ -0,0 +1,216 @@
+#
+# startdate  The date used as the starting date for the hybrid run.
+# ==============================================================================
+
+setenv refcase      g210.G_JRA.v14.gx1v7.01
+setenv refyear      2010
+setenv refmon       01
+setenv refday       01
+setenv reftod       00000
+setenv refdate      ${refyear}-${refmon}-${refday}
+setenv reftimestamp ${refyear}-${refmon}-${refday}-${reftod}
+
+setenv stagedir /glade/scratch/${USER}/${refcase}/rest/${reftimestamp}
+
+# In a hybrid configuration, you can set the startdate to whatever you want.
+# It does not have to match the reference (although changing the month/day seems bad).
+# runtime settings:
+
+setenv start_year    1998
+setenv start_month   01
+setenv start_day     01
+setenv start_tod     00000
+setenv startdate     ${start_year}-${start_month}-${start_day}
+
+# ==============================================================================
+# Make sure the CESM directories exist.
+# VAR is the shell variable name, DIR is the value
+# ==============================================================================
+ 
+foreach VAR ( cesmroot dartroot stagedir )
+   set DIR = `eval echo \${$VAR}`
+   if ( ! -d $DIR ) then
+      echo "ERROR: directory '$DIR' not found"
+      echo " In the setup script check the setting of: $VAR "
+      exit -1
+   endif
+end
+
+# ==============================================================================
+# OSSE/Perfect Model experiments only.
+# If there is an ensemble of POP states to choose from, which one do you want as
+# the truth? There is an argument for picking an instance that will not be part
+# of the ensemble used for the assimilation experiment. 
+
+setenv SingleInstanceRefcase 0
+setenv TRUTHinstance 23
+
+# ==============================================================================
+#
+# stop_option   Units for determining the forecast length between assimilations
+# stop_n        Number of time units in each forecast
+# resubmit      How many job steps to run on continue runs (should be 0 initially)
+
+setenv resubmit     0
+setenv stop_option  ndays
+setenv first_STOP_N 3
+setenv STOP_N       1
+setenv resubmit     0
+
+# ==============================================================================
+# Settings for the data atmosphere
+
+setenv stream_year_align 1998
+setenv stream_year_first 1998
+setenv stream_year_last  2010
+
+setenv short_term_archiver off
+
+# ==============================================================================
+# job settings and machine-specific commands:
+#
+# mach            Computer name
+# ACCOUNT         Project that hours will be charged to
+# queue           can be changed during a series by changing the ${CASE}.run
+# timewall        can be changed during a series by changing the ${CASE}.run
+# ==============================================================================
+
+setenv mach         cheyenne
+setenv ACCOUNT      P93300612
```

Comment:
Frequently we obfuscate the ACCOUNT so as to force them to use their own project number.